src/configuration/sales/braintree.md

```diff
@@ -37,11 +37,11 @@ _Braintree Advanced Settings_
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
 |Vault Title|Website|A descriptive title for your reference that identifies the vault where your customer card information is stored.|
-|Merchant Account ID|Website|The Merchant ID that is to be associated with Braintree transactions from this website. If left blank, the default merchant account from your Braintree account is used.|
+|Merchant Account ID|Store View|The Merchant ID that is to be associated with Braintree transactions from this website. If left blank, the default merchant account from your Braintree account is used.|
```

Comment:
This says `website` in my 2.4.3 instance: 

![image](https://user-images.githubusercontent.com/1128913/142677039-fdca9a86-3346-428b-8c6b-4399813064a5.png)

Reply:
@leslietilling Oh thank you, my mistake 😊